integration/sawtooth_integration/tests/test_intkey_smoke.py

```diff
@@ -138,8 +140,8 @@ def _query_rest_api(suffix='', data=None, headers={}):
 
 class IntkeyTestVerifier:
     def __init__(self,
-                 valid=('cow', 'pig', 'sheep', 'goat', 'horse'),
-                 invalid=('lark', 'thrush', 'jay', 'wren', 'finch'),
+                 valid=('lark', 'thrush', 'jay', 'wren', 'finch'),
```

Comment:
I decided I would rather have the passerine birds be the valid transactions. But I can change it back if you prefer the farm animals. It's _udderly_ inconsequential.